hatrac/model/storage/amazons3.py

```diff
@@ -16,11 +16,15 @@
 import boto
 import boto.s3
 import boto.s3.key
+from boto.s3.connection import S3Connection
 from hatrac.core import BadRequest, coalesce
 import binascii
 import base64
+import logging
 
-class PooledS3Connection (PooledConnection):
+logger = logging.getLogger('hatrac')
```

Comment:
I don't think this logger is being used, and I'd prefer not to have any custom logging like this.  We should raise exceptions and trace them in the rest layer using web.debug etc.